Dockerfile-debian

```diff
@@ -11,10 +11,10 @@ RUN pip install yq==${YQ_VERSION}
 RUN pyinstaller --noconfirm --onefile --log-level DEBUG --clean --distpath /tmp/ $(which yq)
 
 # kubectl binary
-FROM bitnami/kubectl:1.30.6 as kubectl
+FROM bitnami/kubectl:1.33.1 as kubectl
 
 # Main
-FROM node:22.11.0-bullseye-slim
+FROM node:22.16.0-bookworm-slim
```

Comment:
Does `bullseye` and `bookworm` provide the same list of binaries, including versions?
If not, it will affect customer's pipelines.